pinning/hardware/netio.m4

```diff
@@ -1,5 +1,7 @@
-/* add support for RS485 */
-pin(RS485TE_USART0, PD2, OUTPUT)
+ifdef(`conf_MODBUS', `dnl
+  /* add support for RS485 */
+  pin(RS485TE_USART0, PC0, OUTPUT)
```

Comment:
Hmm you're right, maybe better PD3 or PD2 that at least are near TXD/RXD pins. But yes they have to customize that, I have added the comment to menuconfig just for that reason.